packages/services/f-braze-adapter/src/services/BrazeConsumer.js

```diff
@@ -32,9 +34,17 @@ class BrazeConsumer {
         callbacks = {},
         interceptInAppMessages = {},
         interceptInAppMessageClickEvents = {},
-        loggerCallbacks = {},
-        customFilters = []
+        logger = (type, message, data) => console.log(type, message, data),
+        customFilters = [],
+        key: $key
```

Comment:
Is it worth using `consumerKey` for this, given that there only seems to be one usage here, within a template string? The usage of `$key` looks a bit odd to me due to the recurrance of `$` in the tempate string, as well the potential confusion when mixing usage of `$` prefix for member variables.
```suggestion
        key: consumerKey
```

Reply:
Agreed see the confusion there, will make the change